crates/turbopack-css/src/chunk/mod.rs

```diff
@@ -207,16 +209,30 @@ impl Asset for CssChunk {
 
         let path = self_vc.path();
         let chunk_name = path.to_string();
-        let mut code = format!("/* chunk {} */\n", chunk_name.await?);
 
-        let mut writer = WriterWithIndent::new(&mut code);
+        let mut body = "".to_owned();
+        let mut writer = WriterWithIndent::new(&mut body);
+        let mut external_imports = IndexSet::new();
         for entry in this.main_entries.await?.iter() {
             let entry_placeable = CssChunkPlaceableVc::cast_from(entry);
             let entry_content = entry_placeable.as_chunk_item(this.context).content();
 
-            expand_imports(&mut writer, entry_content).await?;
+            for external_import in expand_imports(&mut writer, entry_content).await? {
+                external_imports.insert(external_import.await?.to_owned());
+            }
+        }
+
+        let mut code = format!("/* chunk {} */\n", chunk_name.await?);
+        for external_import in external_imports {
+            writeln!(
+                code,
+                "@import \"{}\";",
+                external_import.replace('"', "\\\"")
```

Comment:
Thats not proper escaping. Please write a `stringify_str` method that converts a `&str` into an CSS string. These are the CSS scanner rules for CSS strings:

```
string1       \"([^\n\r\f\\"]|\\{nl}|{escape})*\"
string2       \'([^\n\r\f\\']|\\{nl}|{escape})*\'
string        {string1}|{string2}
nl            \n|\r\n|\r|\f
h             [0-9a-f]
unicode       \\{h}{1,6}(\r\n|[ \t\r\n\f])?
escape        {unicode}|\\[^\r\n\f0-9a-f]
```

https://www.w3.org/TR/CSS21/grammar.html#scanner

So basically you need to get rid of `\n` `\r` `\f` `\` `"`